### PR TITLE
Add rest of three utils files to Matser

### PR DESCRIPTION
**Pull request by Chris-MS-896**
'/home/steve/cosmos-explorer/src/Utils/AutoPilotUtils.ts',
  '/home/steve/cosmos-explorer/src/Utils/NotificationConsoleUtils.ts',
  '/home/steve/cosmos-explorer/src/Utils/PricingUtils.ts'

### CODE REVIEW

File: src/global.d.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Explorer from "./Explorer/Explorer";
 declare global {
   interface Window {
     authType: AuthType;
-    dataExplorer: Explorer;
+    dataExplorer: any;
```
Comment (southpolesteve) [inline src/global.d.ts:7]: This needs to be `Explorer`

File: src/Utils/NotificationConsoleUtils.ts [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import * as _ from "underscore";
 import { ConsoleDataType } from "../Explorer/Menus/NotificationConsole/NotificationConsoleComponent";
-
+declare global {
```
Comment (southpolesteve) [inline src/Utils/NotificationConsoleUtils.ts:3]: This should not be added. Why is it necessary?

File: src/global.d.ts (hunk 1 above)
Comment (Chris-MS-896) [reply src/global.d.ts]: I have remove all of the files related to Explorer

File: src/Utils/NotificationConsoleUtils.ts (hunk 2 above)
Comment (Chris-MS-896) [reply src/Utils/NotificationConsoleUtils.ts]: I have remove all of the files related to Explorer

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx [hunk 3]
```diff
@@ -99,6 +99,10 @@ export class NotificationConsoleComponent extends React.Component<
     }
   }
 
+  public setElememntRef = (element: HTMLDivElement) => {
```
Comment (ato9000) [inline src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:102]: Should this be HTMLElement

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx [hunk 4]
```diff
@@ -220,12 +224,12 @@ export class NotificationConsoleComponent extends React.Component<
...
 
-  private onFilterSelected(event: React.ChangeEvent<HTMLSelectElement>, option: IDropdownOption): void {
+  private onFilterSelected = (event: React.ChangeEvent<HTMLSelectElement>, option: IDropdownOption): void => {
     this.setState({ selectedFilter: String(option.key) });
-  }
+  };
 
   private getFilteredConsoleData(): ConsoleData[] {
-    let filterType: ConsoleDataType = null;
+    let filterType: Number | null = null;
```
Comment (ato9000) [inline src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:232]: why not 

ConsoleDataType | null

File: src/Utils/NotificationConsoleUtils.ts [hunk 5]
```diff
@@ -21,7 +20,7 @@ export function logConsoleMessage(type: ConsoleDataType, message: string, id?: s
     }
     dataExplorer.logConsoleData({ type: type, date: formattedDate, message: message, id: id });
   }
-  return id;
+  return !id ? "" : id;
```
Comment (ato9000) [inline src/Utils/NotificationConsoleUtils.ts:23]: id || ""

**Review by ato9000 — APPROVED** [review]

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx (hunk 4 above)
Comment (Chris-MS-896) [reply src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:232]: Yes, it works ,I will keep in mind

File: src/Utils/NotificationConsoleUtils.ts (hunk 5 above)
Comment (Chris-MS-896) [reply src/Utils/NotificationConsoleUtils.ts:23]: I changed it

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx [hunk 6]
```diff
@@ -11,10 +11,10 @@ import ErrorBlackIcon from "../../../../images/error_black.svg";
 import infoBubbleIcon from "../../../../images/info-bubble-9x9.svg";
 import InfoIcon from "../../../../images/info_color.svg";
 import ErrorRedIcon from "../../../../images/error_red.svg";
-import LoaderIcon from "../../../../images/circular_loader_black_16x16.gif";
 import ClearIcon from "../../../../images/Clear.svg";
-import ChevronUpIcon from "../../../../images/QueryBuilder/CollapseChevronUp_16x.png";
-import ChevronDownIcon from "../../../../images/QueryBuilder/CollapseChevronDown_16x.png";
+const LoaderIcon = require("../../../../images/circular_loader_black_16x16.gif");
```
Comment (southpolesteve) [inline src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:15]: These are not correct. We shoud be able to use `import`

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx [hunk 7]
```diff
@@ -160,7 +164,7 @@ export class NotificationConsoleComponent extends React.Component<
                 role="combobox"
                 selectedKey={this.state.selectedFilter}
                 options={NotificationConsoleComponent.FilterOptions}
-                onChange={this.onFilterSelected.bind(this)}
+                onChange={() => this.onFilterSelected}
```
Comment (southpolesteve) [inline src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:167]: This is a behavior change. You are not calling `this.onFilterSelected`

`this.onFilterSelected()`

Comment (Chris-MS-896) [reply src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:167]: My fault, I see the onFilterSelected is already arrow function, anyway, I just want to away bind and use arrow function

File: src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx (hunk 6 above)
Comment (Chris-MS-896) [reply src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx:15]: change it back, It can works now with import

**Review by southpolesteve — APPROVED** [review]

**Pull request merged**